packs/classfeatures/omnirange-stabilizers.json

```diff
@@ -24,7 +24,17 @@
             "remaster": false,
             "title": "Pathfinder Guns & Gears"
         },
-        "rules": [],
+        "rules": [
+            {
+                "definition": [
+                    "item:id:{actor|flags.pf2e.trackedItems.weaponInnovation}"
+                ],
+                "key": "AdjustStrike",
+                "mode": "add",
+                "property": "range-increment",
+                "value": 50
+            }
+        ],
```

Comment:
This needs to be predicated on not having the volley trait, and another RE to remove the volley trait if it exists (again, may need to play with priority)

Reply:
Changes made - check when you get a moment